crawler.go

```diff
@@ -48,8 +48,9 @@ func (c crawler) crawl() ([]string, error) {
 
 	// Contracheque
 	log.Printf("Realizando seleção (%s/%s)...", c.month, c.year)
-	if err := c.abreCaixaDialogo(ctx, "contra"); err != nil {
-		log.Fatalf("Erro no setup:%v", err)
+	if err := c.abreCaixaDialogo(ctx, "contracheque"); err != nil {
+		log.Printf("Erro no setup:%v", err)
```

Comment:
```suggestion
		fmt.Fprintf(os.Stderr"Erro no setup: %v", err)
```

Poderia ser mais descritivo na mensagem de erro, por favor? Quanto mais a gente fornecer de informação, melhor.